changelogs/fragments/83715-raise-on-invalid-collection-entry.yml

```
@@ -0,0 +1,2 @@
+bugfixes:
+  - ansible-galaxy collection install - raise errors on invalid collection entry in requirements.yml (https://github.com/ansible/ansible/issues/83699).
```

```suggestion
  - ansible-galaxy collection install|download|verify - raise errors on invalid collection entry in requirements.yml (https://github.com/ansible/ansible/issues/83699).
```

---

Maybe also highlight things:
```suggestion
  - ``ansible-galaxy collection install|download|verify`` - raise errors on invalid collection entry in :file:`requirements.yml` (https://github.com/ansible/ansible/issues/83699).
```

---

I think `:file:` is a Sphinx extension, and not something supported natively by docutils.

---

Fair. Although, I don't think GH even uses docutils. Their parser is really dumb.